src/Tinkoff.ISA.Providers.Jira/JiraProviderSettings.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Tinkoff.ISA.Providers.Jira
+{
+    public class JiraProviderSettings
+    {
+        [Required]
```

Comment:
how we check this required field?

Reply:
This check is enabled here:
```
serviceCollection.AddOptions<JiraProviderSettings>()
                .Bind(jiraSection)
                .ValidateDataAnnotations();
```
Validation runs when the options instance is created.

Reply:
But are we going to remove IOption dependency ?